test/support/coronavirus_landing_page_steps.rb

```diff
@@ -65,6 +69,14 @@ def then_i_can_see_the_live_stream_section_with_date_and_time
     assert page.has_text?("5:00pm")
   end
 
+  def then_i_can_see_the_ask_a_question_section
+    assert page.has_link?("Ask a question at the next press conference", href: "https://www.gov.uk")
```

Comment:
Should this be "https://www.gov.uk/ask" ?

Reply:
We've cleared this up on slack. I didn't realise the value is being pulled in from the fixture.